### ISSUE TITLE
on v3 can not get origURL[🐛v3] 

### ISSUE THREAD

**Issue by tianlu1677** [react-native-image-picker/react-native-image-picker#1536]
on v3 I can not get origURL for video

**Comment by Johan-dutoit** [comment]
Can you show code, context, etc..

**Comment by Hamza6244** [comment]
@Johan-dutoit as in version 3 you have removed **path**  response and just returning **uri** for both android and iOS then now how can access file and use compression on that file because currently i'm unable to do compression etc on response.uri

**Comment by tianlu1677** [comment]
on v2.3.3
```
ImagePicker.launchImageLibrary(
        {
          mediaType: 'video',
          videoQuality: 'low',
        },
        async response => {
          if (response.didCancel) {
            return;
          }
          console.log('response', response);
          // return
          const video = response;
          props.removeAllPhoto();
          let videoSourceContent = {
            uri: video.origURL
            // height: video.height,
            // width: video.width,
            // duration: video.duration,
          };      
        } 
```
I can get origURL and uri.

On v3.1.0. I can only get uri. uri is compressed video uri, so i can not get origin video path.
```
launchImageLibrary(
        {
          mediaType: 'video',
          videoQuality: 'low',
        },
        async response => {
          if (response.didCancel) {
            return;
          }
          console.log('response', response);
          // return
          const video = response;
          props.removeAllPhoto();
          let videoSourceContent = {
            uri: video.uri, 
            // height: video.height,
            // width: video.width,
            // duration: video.duration,
          };
         
        }
```
@Johan-dutoit

**Comment by Hamza6244** [comment]
@tianlu1677 So you mean in v3.1.0 we have no need to compress video manually and it when we will get uri of video then it will return compress video right?

**Comment by ravirajn22** [comment]
We don't compress videos from what I know (iOS might by default I need to check on that) and you can use the returned URI to read the file. Read, compress and save where ever you want. If you tell about the library or code you use to compress that will help to get a clear picture.

**Comment by SirTitch** [comment]
I'm getting something similar where I was using another library to get a video thumbnail using response.uri, and now its not working any more since the update. Any idea how to get a fix?

**Comment by ravirajn22** [comment]
@SirTitch post details when you comment in issues, like the other library used to generate video thumbnail. Is this issue happening in iOS or android or both. When you can upload, use it in Image component etc. Why is that some third party libraries does not work with this uri. Tell all your third party libraries you use with uri, I will check.

**Comment by Hamza6244** [comment]
@ravirajn22 How all libraries almost working with uri in case of iOS but in android, it's not working and it will be better if you return path as well with uri in android

**Comment by ravirajn22** [comment]
List library names

**Comment by Hamza6244** [comment]
@ravirajn22 
**

1. 'react-native-video-processing'
2. 'rn-media-meta'
3. 'react-native-thumbnail'

**

**Comment by tianfan** [comment]
Yes. please put back the response.origURL. So the response.uri is a local identifier, it doesn't work if you want to copy (with RNFS) the image asset to the app's document directory. I don't understand why it got removed at the first place.
